src/prefect/client/orchestration.py

```diff
@@ -2694,6 +2694,89 @@ async def release_concurrency_slots(
             },
         )
 
+    async def create_concurrency_limit_global(
```

Comment:
Could we rename these to be have `global` before `concurrency`? `create_global_concurrency_limit`, `update_global_concurrency_limit`, etc. I think that would make it more intuitive.

Reply:
sure, i'll change this :)